docs/validators/staking/staking-smart-contract.md

```diff
@@ -93,7 +93,7 @@ Note that:
 
 - `45e7131ba....294812f004` is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
 - `ecf6fdbf5....70f1d251f7` is the BLS key of the second node, represented as a 192-character-long hexadecimal string;
-- no reserved placeholder is needed, as opposed to the staking transaction (see above)
+- no result of signing the address of the sender of the transaction is needed, as opposed to the staking transaction (see above)
```

Comment:
I think it's safe to shorten this to "BLS signature" everywhere. It should be clear from context what this means.